README.md

```diff
@@ -7,3 +7,136 @@
 [![codecov](https://codecov.io/gh/spacetelescope/stcal/branch/main/graph/badge.svg?token=C1LO00W9CZ)](https://codecov.io/gh/spacetelescope/stcal)
 
 STScI Calibration algorithms and tools.
+
+![STScI Logo](docs/_static/stsci_logo.png)
+
+**STCAL requires Python 3.8 or above and a C compiler for dependencies.**
```

Comment:
Astropy is dropping support for Python 3.8 in the next release (coming out some time in may). Let's move to Python 3.9 as a min dependency.

Reply:
I made this change.